src/tester.rs

```diff
@@ -445,35 +463,30 @@ fn run_tests(
         let failures = failures.clone();
         let inner = inner.clone();
         pool.execute(move || {
-            if inner.test_threads == 1 {
+            let single_threaded = inner.test_threads == 1;
```

Comment:
I'm not sure this assignment is useful given that it's only read from once.

Reply:
It's passed into `write_ignored` because the output differs depending on whether or not we're in a single-threaded context.

Reply:
Ah, I see! I think it might fit more with the other code to pass `inner: Arc<LangTesterPooler>` as an argument rather than `single_threaded`?

Reply:
Fixed: 46ce135cdb6444e9f261fc5077f0db01bca99255